### DISCUSSION TITLE
Copilot`s message hides code

### DISCUSSION THREAD

**Discussion by vernik03** [community/community#17567] [Copilot Conversations]
Message `TAB to accept | Alt+. to next` overlaps my code in Microsoft Visual Studio 2022.

**Comment by L-Mustang** [comment]
Same here. The popup continues to block any code written behind it.
Visual Studio 2022 17.2.3. 
![image](https://user-images.githubusercontent.com/33723142/171414325-90cba7ac-f344-4b75-a0da-f060943d10fb.png)

**Reply by ViacheslavBezditnyi** [reply]
Yes, the same thing after updating Visual Studio to 17.2.3.

**Reply by iliaakm** [reply]
Same with 17.2.4

**Reply by cpgames** [reply]
And in 17.2.5
